### PR TITLE
Adding CvTs

### PR DESCRIPTION
**Pull request by LucyV4**
Adds all of the CvTs from https://arxiv.org/abs/2103.15808 to the models and adds this paper as a reference to the new and existing CvTs

### CODE REVIEW

**Comment by KartikP** [comment]
Looks like `cvt_cvt_w24_384_in22k_finetuned_in1k_4` ran out of memory. Going to open/close PR to retrigger unit test.

**Comment by LucyV4** [comment]
CvT W24 is already on brain score, so that one does not have to be runned necessarily. I only added the bibtex of the paper for that one. The CvT 13's and 21's are new

File: brainscore_vision/models/cvt_cvt_21_384_in1k_4/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from brainscore_vision import model_registry
+from brainscore_vision.model_helpers.brain_transformation import ModelCommitment
+from .model import get_model, get_layers
+
+
+model_registry['cvt_cvt-13-384-in1k_4'] = \
+    lambda: ModelCommitment(identifier='cvt_cvt-13-384-in1k_4',
+                            activations_model=get_model('cvt_cvt-13-384-in1k_4'),
+                            layers=get_layers('cvt_cvt-13-384-in1k_4'))
```
Comment (KartikP) [inline brainscore_vision/models/cvt_cvt_21_384_in1k_4/__init__.py:9]: It appears that your `__init__.py` file for your `cvt_cvt_21_384_in1k_4` and `cvt_cvt_13_384_in_1k_4` are swapped and creating inconsistencies in the model_identifier for everything else in the respective directories.

Should be an easy fix and once that's done, I think this should pass.

**Review by KartikP — CHANGES_REQUESTED** [review]
`cvt_cvt_21_384_in1k_4` and `cvt_cvt_13_384_in_1k_4` init files are swapped.

**Comment by KartikP** [comment]
`cvt_cvt-w24-384-in22k_finetuned-in1k_4` still causing out of memory issues. @LucyV4, could you remove the CVT W24 model from your PR. Then we can merge and score it.

**Comment by LucyV4** [comment]
Removed the bibtex from the W24 model

**Comment by LucyV4** [comment]
I see that the check failed because of my branch name, I think closing this and then making a new PR from the master branch of the fork is the best course of action?

**Comment by KartikP** [comment]
@LucyV4 Hi Lucy, you're right but don't do anything just yet. There's one thing I want to clarify soon with the other developers regarding some private models that has the same model_identifiers (i.e., someone else has submitted the suite of CVT models) that is causing some issues. I'll get back to you today about what we'll do.

**Comment by LucyV4** [comment]
The private submission was me as well, I am writing a paper on vision models, illusions and brain scores. I found that the private submission had around 20 more tests and I did not want to upload all 6 models seperately

**Comment by KartikP** [comment]
@LucyV4 To explain the situation at hand, earlier in the year, another user had submitted `cvt_cvt-{13,21}-{224,384}-in1k/{-in1k}` models. Essentially with the same identifiers as you did but they listed them as private. What this is now causing us is a conflict in our database with duplicate model_identifiers. Because this model is private, it also does not show up in the `models` directory.

In the future, we plan to have namespaces and keep all private models in the user-specific namespace, but we do not yet have this implemented.

For now, I'm going to ask you to append a unique identifier to each of your model identifiers. For example, `cvt_cvt-13-384-in1k_4` will become `cvt_cvt-13-384-in1k_4_lucyV4` or `keemink-lab_cvt_cvt-13-384-in1k_4`.

Once that's done, I'll let tests run again, and then merge this for you so you can get scores for your models.

**Pull request merged**